spec/support/network_helpers.rb

```diff
@@ -0,0 +1,12 @@
+module NetworkHelpers
+  def available_endpoint
+    "0.0.0.0:#{available_port}"
+  end
+
+  def available_port
```

Comment:
This is a clever way of "allocating" an address/port for the integration test. Just so it's clear to anyone who sees this, it might be a good idea to add a comment describing the purpose/use of these functions.

Reply:
Will do!

Reply:
Added documentation to new helper methods.